### DISCUSSION TITLE
2022-08-29-Release-Preview-Bullseye

### DISCUSSION THREAD

**Discussion by mgrouch** [bareboat-necessities/lysmarine_gen#224] [General]
2022-08-29-Release-Preview-Bullseye

This is a pre-release

64-bit, Bullseye

Download from: 

https://cloudsmith.io/~bbn-projects/repos/bbn-repo/packages/?q=lysmarine-bbn-bullseye

<hr /><em>This discussion was created from the release <a href='https://github.com/bareboat-necessities/lysmarine_gen/releases/tag/bullseye-2022-08-29'>2022-08-29-Release-Preview-Bullseye</a>.</em>

**Comment by mgrouch** [comment]
Few things and observations that are different in Bullseye:

1. Screen resolution switch is quite different from Buster with kms driver
2. You no longer need to reboot after switch from audio jack to HDMI audio. You can do it via settings UI, at runtime
3. Caprine notifications need to be tested 
4. Hamfax is not available (at least yet) due to drop of qt4 in favor of qt5
5. Keyboard language switch needs more testing
6. Geoclue and gnome weather application seem having issue
7. SignalK iridium sbd plugin doesn't compile and was dropped from distro image
8. NmeaConverter_pi is not included

Fair Winds!

**Comment by wsturner88** [comment]
Downloading now.. I will try it out later this afternoon and also confirm it the OpenCPN issue is resolved.
Thanks for cranking this out.
